apps/webapp/app/v3/failedTaskRun.server.ts

```diff
@@ -40,7 +39,13 @@ export class FailedTaskRunService extends BaseService {
     // No more retries, we need to fail the task run
     logger.debug("[FailedTaskRunService] Failing task run", { taskRun, completion });
 
-    await marqs?.acknowledgeMessage(taskRun.id);
+    const finalizeService = new FinalizeTaskRunService();
```

Comment:
`BaseService` already has a constructor that takes a `PrismaClientOrTransaction`, so you should probably just use that instead of passing the transaction as an option to the call method.